### PR TITLE
What the sql - Recursive

### PR DESCRIPTION
**Pull request by ddrscott**

### CODE REVIEW

File: source/_posts/2017-03-15-what-the-sql-recursive.markdown [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+published: true
+---
+
+{% img featured /images/i_heart_recursion.png 1804 1195 'What the SQL?!? Recursive' %}
+
+# What the SQL?!? Recursive
+
+Today's "What the SQL?!?" features the keyword `RECURSIVE`. This clause allows
+us to elegantly select results from the previous results from previous results from
+previous results...
```
Comment (lrakhman) [inline source/_posts/2017-03-15-what-the-sql-recursive.markdown]: Should be: `This clause allows us to elegantly select results from the previous results from THE previous results from THE...`

File: source/_posts/2017-03-15-what-the-sql-recursive.markdown [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+(10 rows)
+```
+
+If you want to see the results for a high number, update `i < 10` to a higher
+value. If you go above `i < 793`, Postgres gives up and returns `Nan` which means
+`Not a number` which means the computed value is larger than your computer can
+handle and still treat like a number. Sorry, get a new computer or work with
+numbers less than 166 digits long.
+
+# An Real World Example with Hierarchal Data
```
Comment (lrakhman) [inline source/_posts/2017-03-15-what-the-sql-recursive.markdown]: A, not 'An' Real World

Comment (lrakhman) [inline source/_posts/2017-03-15-what-the-sql-recursive.markdown]: A Real World Example with * Hierarchical* Data

File: source/_posts/2017-03-15-what-the-sql-recursive.markdown [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+Fibonacci sequence is nice and all, but you have real data concerns. You're
+thinking, "Show me the DATA!". So here's the data...
+
+```sql
+-- Build `sample_people` table
+CREATE TABLE sample_people AS
+  SELECT
+    column1::int     AS id,
+    column2::varchar AS name,
+    column3::int AS parent_id
```
Comment (lrakhman) [inline source/_posts/2017-03-15-what-the-sql-recursive.markdown]: This column should line up with the previous two (on the word 'AS')

**Comment by lrakhman** [comment]
It's very hard to read the recursion_sql png. In fact, I couldn't read most of it :(. Otherwise, this LGTM!! 🙇‍♀️

**Comment by ddrscott** [comment]
Thanks Lana!

**Pull request closed without merging**